test/e2e/wmco_test.go

```diff
@@ -131,3 +146,68 @@ func testWindowsNodeDeletion(t *testing.T) {
 	}
 	defer testCtx.Cleanup()
 }
+
+// testWMCValidation tests if validations of the fields of WindowsMachineConfigs CRD are working as expected
+func testWMCValidation(t *testing.T) {
+	var wmcReplicasFieldValidationTests = []struct {
+		name                       string
+		isReplicasFieldPresent     bool
+		replicasFieldValue         int
+		isTestExpectedToThrowError bool
+		expectedErrorInTest        string
+	}{
+		// We are only checking negative test cases here, positive test cases would check if custom resource is
+		// getting created as expected and they are handled elsewhere
```

Comment:
Where exactly are they handled? I think it'd be better if you mention the test name or have this comment at the testWMCValidation function level

Reply:
Done